models/app.model.js

```diff
@@ -19,3 +19,14 @@ exports.findArticlesById = (id) => {
             }
         })
 }
+
+exports.findArticleComments = (id) => {
+    return db.query(`
+        SELECT comment_id, comments.votes, comments.created_at,
+        comments.author, comments.body, comments.article_id
+        FROM comments
+        JOIN articles on comments.article_id = articles.article_id
```

Comment:
You are going to need a left join here. This is because if there are no comments for that ID it will just be ignored.